web/js/map-wms-base.js

```diff
@@ -0,0 +1,29 @@
+// Functionality required by all OpenLayers WMS base maps
+
+fixmystreet.maps.setup_wms_base_map = function() {
+    fixmystreet.map_type = OpenLayers.Layer.WMS;
+
+    fixmystreet.map_options = {
+        maxExtent: this.layer_bounds,
+        units: 'm',
+    };
+
+    fixmystreet.layer_options = [];
+    $.each(fixmystreet.wms_config.layer_names, function(i, v) {
+        fixmystreet.layer_options.push({
```

Comment:
It seems odd to have a template creating the wms_config data object, and then having the JS having to do lots of work here to use it? Why is the JS here not `fixmystreet.layer_options.push(fixmystreet.wms_config)` or at least trying to map it across with as little work as possible?